### PR TITLE
Create, update, delete words for admin

### PR DESCRIPTION
**Pull request by NguyenTanDuc**

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -36,4 +36,15 @@ def log_out
...
     @current_user = nil
   end
+
+  def admin_user
+    redirect_to log_in_path unless current_user.level == 1
+  end
+
+  def logged_in_user
+    unless logged_in?
+      flash[:danger] = "Please log in."
```
Comment (leduytungframgia) [inline app/helpers/sessions_helper.rb]: i18n

File: app/views/admin/words/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <th>ID</th>
+        <th>English mean</th>
+        <th>Action</th>
+      </tr>
+      <% @words.each do |word| %>
+        <tr>
+          <td><%= word.id %></td>
+          <td><%= word.content %></td>
+          <td>
+            <%= link_to "Edit", edit_word_path(word.id) %>
```
Comment (khanhkhanh) [inline app/views/admin/words/index.html.erb]: i18n

File: app/views/admin/words/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <%= link_to "Delete", word, method: :delete,
```
Comment (khanhkhanh) [inline app/views/admin/words/index.html.erb]: i18n

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (khanhkhanh) [inline app/helpers/sessions_helper.rb]: i18n

File: app/controllers/admin/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+  def index
+    @words = Word.paginate page: params[:page], per_page: 20
+    @categories = Category.all
+  end
+
+  def new
+    @word = Word.new
+    @categories = Category.all
+    4.times do
```
Comment (dieunb) [inline app/controllers/admin/words_controller.rb]: should 
```
4.times {@word.answers.build}
```

File: config/routes.rb [hunk 5]
```diff
@@ -7,8 +7,11 @@
   delete 'log_out' => 'sessions#destroy'
 
   resources :users, except: [:index]
-  resources :words
+  resources :words, only: [:index]
   resources :categories
   resources :lessons, except: [:edit, :update]
 
+  namespace :admin do
+    resources :words
```
Comment (dieunb) [inline config/routes.rb]: root page???

File: config/routes.rb [hunk 6]
```diff
@@ -7,8 +7,11 @@
   delete 'log_out' => 'sessions#destroy'
 
   resources :users, except: [:index]
-  resources :words
+  resources :words, only: [:index]
```
Comment (dieunb) [inline config/routes.rb]: move to admin namespace

File: config/routes.rb [hunk 7]
```diff
@@ -7,8 +7,11 @@
...
   resources :categories
```
Comment (dieunb) [inline config/routes.rb]: move to admin namespace

File: app/helpers/sessions_helper.rb [hunk 8]
```diff
@@ -36,4 +36,15 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def admin_user
+    redirect_to log_in_path unless current_user.level == 1
```
Comment (dieunb) [inline app/helpers/sessions_helper.rb]: you should create a function is called `is_admin` at user model. After that, you can check user is admin by `user.is_admin?`

File: app/views/shared/_error_messages.html.erb [hunk 9]
```diff
@@ -1,6 +1,6 @@
 <% if object.errors.any? %>
   <div class="alert alert-danger">
-    You have <%= pluralize(object.errors.count, I18n.t(:errors)) %> in this form.
+  You have <%= pluralize(object.errors.count, t(:errors)) %> in this form.
```
Comment (dieunb) [inline app/views/shared/_error_messages.html.erb]: i18n

File: app/models/user.rb [hunk 10]
```diff
@@ -44,4 +44,8 @@ def authenticated? remember_token
   def forget
     update_attribute :remember_digest, nil
```
Comment (dieunb) [inline app/models/user.rb]: try `update_attributes!` and compare with `update_attribute`.

**Pull request merged**